src/os/bluestore/kv.h

```diff
@@ -8,6 +8,7 @@
 
 // some key encoding helpers
 void _key_encode_u32(uint32_t u, std::string *key);
+void _key_encode_u32(uint32_t u, std::string *key, size_t pos);
```

Comment:
Welcome back, Igor.

I'll rebase once https://github.com/ceph/ceph/pull/12651 is merged.